packages/carbon-web-components/src/components/pagination/pagination.scss

```diff
@@ -94,6 +94,11 @@ $css--plex: true !default;
   align-items: center;
   height: 100%;
   position: relative;
+
+  .#{$prefix}--visually-hidden {
```

Comment:
<img width="747" alt="Screen Shot 2023-02-20 at 10 41 08 AM" src="https://user-images.githubusercontent.com/20210594/220173663-68893969-24e8-4c6e-b7ca-88a77df44643.png">

there's some extra styles in the visually-hidden utility: https://github.com/carbon-design-system/carbon/blob/main/packages/styles/scss/utilities/_visually-hidden.scss#L13 that would help remove that margin spacing

Reply:
update you should be able just to add `@use '@carbon/styles/scss/utilities' as *;` to the top and remove the specific styles